openmc/lattice.py

```diff
@@ -93,9 +93,8 @@ def universes(self):
     @id.setter
     def id(self, lattice_id):
         if lattice_id is None:
-            global AUTO_UNIVERSE_ID
```

Comment:
Do you want to remove this `global` declaration in every other class' `id` setter?